plugins/out_kinesis_streams/kinesis.c

```diff
@@ -440,6 +440,12 @@ static struct flb_config_map config_map[] = {
      "Specify a custom endpoint for the Kinesis API"
     },
 
+    {
+     FLB_CONFIG_MAP_INT, "port", "443",
+     0, FLB_FALSE, 0,
```

Comment:
If there is a default value here, you should specify set_property part as `FLB_TRUE`.

See: https://github.com/fluent/fluent-bit/blob/master/include/fluent-bit/flb_config_map.h#L75